amp/parse.go

```diff
@@ -20,7 +20,11 @@ type Params struct {
 	Slot            string
 	StoredRequestID string
 	Timeout         *uint64
-	Targeting       string
+
```

Comment:
Please remove the line break and alphabetize the list with the new fields added by this PR.